src/CampaignAdmin.php

```diff
@@ -588,12 +589,26 @@ public function save($data, $form)
             $record = ChangeSet::create();
         }
 
-        // save form data into record
-        $form->saveInto($record, true);
-        $record->write();
-        $this->extend('onAfterSave', $record);
-
-        $message = _t('LeftAndMain.SAVEDUP', 'Saved.');
+        $hasExistingName = Changeset::get()
+            ->filter('Name:case', $data['Name'])
```

Comment:
I would opt for case-insensitivity when filtering for a duplicate.
Having an entry `My case` and allowing for `My Case` to also be created is going to be confusing.

Was this discussed to have it case-sensitive?